widget/select_test.go

```diff
@@ -151,8 +150,8 @@ func TestSelect_Tapped(t *testing.T) {
 	pos := fyne.CurrentApp().Driver().AbsolutePositionForObject(over)
 
 	cont := over.(*PopUp).Content
-	assert.Equal(t, cont.Position().X, pos.X+theme.Padding())
```

Comment:
Here it seems like we are testing that the select drop down no longer has padding - is that correct?

Reply:
Yes, because it is a menu pop-up and these are no longer padded because the items already have padding now.